source/_integrations/unifi.markdown

```diff
@@ -62,7 +62,15 @@ It is recommended that you run the UniFi Network application in a dedicated virt
 
 ## Button
 
-Provides the ability to restart a UniFi device. This feature requires admin privileges.
+Button entities requires admin privileges to use.
+
+### Power cycle PoE
+
+Use this button entity to cycle power output of a PoE port to cause a connected device to restart.
+
+### Restart UniFi device
+
+Use this button entity to restart a select UniFi device.
```

Comment:
```suggestion
Use the **Restart UniFi device** button entity to restart the entire UniFi device. This will cause all connected PoE devices to restart.
```

Reply:
```suggestion
Use the **Restart UniFi device** button entity to restart the entire UniFi device. In case the device is a PoE switch PoE supply is not affected.
```